package/src/values/types.ts

```diff
@@ -23,6 +23,10 @@ export interface Value<T = number> extends ReadonlyValue<T> {
    * Get/sets the animation controlling the value
    * */
   animation: Animation | undefined;
+  /**
+   * Cancel running animation if there is one.
+   */
+  cancelAnimation: () => void;
```

Comment:
I'm thinking since we have automatic cancellation when a value is set that this is maybe not necessary?

Reply:
indeed, super nice